packages/esm-service-queues-app/src/queue-table/cells/queue-table-patient-identifier-cell.component.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { type PatientIdentifierColumnConfig } from '../../config-schema';
+import { type QueueEntry, type QueueTableColumnFunction, type QueueTableCellComponentProps } from '../../types';
+
+export const queueTablePatientIdentifierColumn: QueueTableColumnFunction = (
+  key,
+  header,
+  config: PatientIdentifierColumnConfig,
+) => {
+  const { identifierType } = config;
+
+  const getPatientId = (queueEntry: QueueEntry) => {
+    for (const identifier of queueEntry.patient.identifiers) {
+      if (identifier.identifierType?.uuid == identifierType) {
```

Comment:
We likely want to check beyond this.  If a patient has > 1 identifier of the same type, then we should prioritize a) any that are "preferred = true" first, then likely the most recently created second.  Ideally the API could give you this, but otherwise should be done here.